### PR TITLE
Renaming apis from using metadatas to metadata

### PR DESCRIPTION
**Pull request by zhljen**

### CODE REVIEW

File: metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
      *
      * @param uris list of uris.
      */
-    default void deleteDataMetadatas(List<String> uris) {
+    default void deleteDataMetadata(List<String> uris) {
```
Comment (ajoymajumdar) [inline metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java]: Why this change from plural to singular? This method is deleting mutiple data metadatas.

Comment (zhljen) [reply metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java]: the plural of metadata is metadata..

Comment (ajoymajumdar) [reply metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java]: Do we really need this change?

Comment (zhljen) [reply metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/UserMetadataService.java]: I separated out as a new pull from the previous one. We should fix this name.
